Scenarios/UseAzureKeyVault.md

```diff
@@ -1,11 +1,12 @@
 # #7 Use Azure KeyVault for secrets with AL-Go
-*Prerequisites: A completed [scenario 6](UpdateAlGoSystemFiles.md), an Azure KeyVault and you will need to follow the guidelines on how to connect to an Azure KeyVault as specified here: https://docs.microsoft.com/en-us/azure/developer/github/github-key-vault. Add your KeyVault name to the the JSON construct from this walkthrough (using **“keyVaultName” : “{your keyvault name}”**) and add this JSON construct as a repository secret called AZURE_CREDENTIALS. You can also specify the KeyVault name in the AL-Go settings file if you do not wait to mess with the JSON construct.*
+*Prerequisites: A completed [scenario 6](UpdateAlGoSystemFiles.md), an Azure KeyVault and you will need to follow the guidelines on how to connect to an Azure KeyVault as specified here: [https://docs.microsoft.com/en-us/azure/developer/github/github-key-vault](https://docs.microsoft.com/en-us/azure/developer/github/github-key-vault). Add your KeyVault name to the the JSON construct from this walkthrough (using **“keyVaultName” : “{your keyvault name}”**) and add this JSON construct as a repository secret called AZURE_CREDENTIALS. You can also specify the KeyVault name in the AL-Go settings file if you do not wait to mess with the JSON construct.*
```

Comment:
Isn't `doc.microsoft.com` currently migrated to `learn.microsoft.com`? Or was it `go.microsoft.com`? 🤔

Reply:
Correct - https://learn.microsoft.com/azure/developer/github/github-key-vault would be better

Reply:
made the change